crimsobot/cogs/games.py

```diff
@@ -143,9 +143,13 @@ def check(msg: discord.Message) -> bool:
             'you CANNOT be serious',
             'sure? how would i know?',
             'what heck',
-            await m.async_wrap(self.bot, m.crimso)
+            'random_response',  # leave this alone
         ]
 
+        response = random.choice(answer_list)
+        if response is 'random_response':
```

Comment:
```suggestion
        if response == 'random_response':
```
mind changing it to this? string comparisons get wonky in python sometimes, this can sometimes return false due to it being a different instance